docs/src/main/asciidoc/deploying-native-executable-openshift.adoc

```diff
@@ -0,0 +1,196 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+
+[id="deploy-openshift-native-executable"]
```

Comment:
Fix ID value to match the filename.
```suggestion
[id="deploying-native-executable-openshift"]
```
Given that gerunds are discouraged in the Quarkus style guide, should we consider dropping the `ing` here in the filename and title? Although in saying that, all the other deploy topics use a gerund in the filename. 

A URL redirect would also be needed over in the [quarkusio](https://github.com/quarkusio/quarkusio.github.io/tree/develop/_redirects) repo, if the file already exists.